client/state/data-layer/wpcom/sites/gutenberg/test/index.js

```diff
@@ -0,0 +1,30 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import { setType } from '../';
+import { http } from 'state/data-layer/wpcom-http/actions';
+import { setSelectedEditor } from 'state/selected-editor/actions';
+
+describe( 'setType()', () => {
+	test( 'should dispatch a http request', () => {
+		const action = setSelectedEditor( 123, 'gutenberg' );
+		const output = setType( action );
+		expect( output ).toEqual(
+			http(
+				{
+					path: `/sites/123/gutenberg`,
+					method: 'POST',
+					apiNamespace: 'wpcom/v2',
+					query: {
+						editor: 'gutenberg',
+						http_envelope: 1,
+					},
+					body: {},
+				},
+				action
+			)
```

Comment:
at this point the test is doing little more than duplicating the code - we're making sure that people don't change code vs. testing behaviors

personally I tend to skip this test because it's not adding much value to us. we can consider adding some integration tests to make sure the call goes out but meh, make a judgment call on value vs. maintenance burden

Reply:
Removed!